frontend/src/static/js/api.js

```diff
@@ -70,9 +70,8 @@ window.api = {
           ];
 
           obj.searchPath = searchParams.join('_').toLowerCase();
-          obj.repoPath = `${repo.displayName}`;
           obj.repoName = `${repo.displayName}`;
-          obj.location = `${repo.location.location}`;
+          obj.location = `https:/github.com/${repo.location.organization}/${repo.location.repoName}/tree/${repo.branch}`;
```

Comment:
I don't think it is a good idea to set `obj.location` to be related to GitHub only.  We also support analyzing local repos. This will break the location of local repo. Instead, edit the function of getting the github link.

Reply:
Okay noted am on it.

Reply:
Sorry, I didn't see the change, my bad.